w3-retrieval.md

```diff
@@ -0,0 +1,131 @@
+# Retrieval
+
+## Editors
+
+- [Alan Shaw](https://github.com/alanshaw), [Storacha](https://storacha.network/)
+
+## Authors
+
+- [Alan Shaw](https://github.com/alanshaw), [Storacha](https://storacha.network/)
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Abstract
+
+This is a specification that defines UCAN capabilities for authorizing retrieval of resources.
+
+## Introduction
+
+In the Storacha Network data retrievals are authorized by UCAN to allow for strict egress accounting. UCAN authorized retrievals do not prove delivery of data but they can be used to prove that data was requested by an authorized agent.
+
+**It is intended that this specification is used in conjunction with [HTTP Header UCAN Invocation](./http-header-ucan-invocation.md) to allow resources to be served in the same request they are authorized in.**
+
+## Content Retrieve
+
+An authorized agent MAY invoke the `space/content/retrieve` capability on the space subject to retrieve blob bytes.
+
+### Invocation Example
+
+Shown Invocation example illustrates Alice requesting to retrieve 1,636 bytes from a blob stored in her space.
+
+> ℹ️ Note: we use `// "/": "bafy..` comments to denote CID of the parent object.
+
+```jsonc
+{ // "/": "bafy..retrieve"
+  "iss": "did:key:zAlice",
+  "aud": "did:key:zStorageNode",
+  "att": [{
+    "can": "space/content/retrieve",
+    "with": "did:key:zAliceSpace",
+    "nb": {
+      "blob": {
+        // multihash of the blob to retrieve data from as byte array
+        "digest": { "/": { "bytes": "mEi...sfKg" } }
+      },
```

Comment:
any reason we don't make this just `digest`?
```suggestion
      // multihash of the blob to retrieve data from as byte array
      "digest": { "/": { "bytes": "mEi...sfKg" } },
```

in `go-libstoracha` a `blob` is usually a `digest` and a `size`.